spec/models/author_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'rails_helper'
+
+describe Author, type: :model, redis: :true do
+	it { should have_many(:unit_templates) }
+	it { is_expected.to callback(:delete_relevant_caches).after(:commit) }
+
+	DEFAULT_AVATAR_TEST_URL = 'https://assets.quill.org/images/authors/placeholder.png'
+
+	describe 'default avatar url' do
+		it 'should have the correct default avatar url' do
+			expect(Author::DEFAULT_AVATAR_URL).to eq(DEFAULT_AVATAR_TEST_URL)
+		end
+	end
+
+	describe 'delete relevant caches after commit' do
+		let(:author) { create(:author) }
+		let(:template) { create(:unit_template, author: author) }
+
+		it 'should clear the unit templates' do
+			author.destroy
```

Comment:
Let's call `template.get_cached_serialized_unit_template` before calling `author.destroy` to make sure that the template is actually cached first. We can also write an expectation that there exists the associated key to make sure that it exists before clearing it.

Reply:
makes sense